### ISSUE TITLE
🐛 Can not call API when Prefork=true for v2.4.0

### ISSUE THREAD

**Issue by Dikman** [gofiber/fiber#1141]
**Fiber version**
v2.4.0

**Issue description**
I can not call API when Prefork=true
```
$ curl -v 'http://127.0.0.1:3000/echo'
*   Trying 127.0.0.1:3000...
* Connected to 127.0.0.1 (127.0.0.1) port 3000 (#0)
> GET /echo HTTP/1.1
> Host: 127.0.0.1:3000
> User-Agent: curl/7.71.1
> Accept: */*
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 404 Not Found
< Date: Sun, 31 Jan 2021 20:52:28 GMT
< Content-Type: text/plain; charset=utf-8
< Content-Length: 16
< 
Cannot GET /echo* Connection #0 to host 127.0.0.1 left intact 
```

**Code snippet**

main.go
```go
package main

import (
	"log"

	"github.com/gofiber/fiber/v2"
)

func main() {
	app := fiber.New(fiber.Config{
		Prefork: true,
	})

	app.Get("/echo", func(c *fiber.Ctx) error {
		return c.SendString("Hello, World 👋!")
	})

	log.Fatal(app.Listen(":3000"))
}
```

go.mod
```go
module main

go 1.15

require github.com/gofiber/fiber/v2 v2.4.0
```

**PS**
If I removed the line with "Prefork: true", all will be work.

**Comment by ReneWerner87** [comment]
thanks for the report

are you using docker ? or native golang on your system ?
which operating system are you using and in which shell do you start the server ?

**Comment by irgeek** [comment]
I've just run into the same issue.


> are you using docker ? 

No.

> or native golang on your system ?

Yes

> which operating system are you using

I'm running on macOS Catalina (10.15.7)

> in which shell do you start the server ?

```
localhost golang-fiber-prefork $ echo $SHELL
/bin/bash
localhost golang-fiber-prefork $ echo $BASH_VERSION
3.2.57(1)-release
```

I also tried `csh`, `sh` and `dash` but they had the same result -- a single-process server responds correctly, but when prefork is enabled every request results in a 404.

**Comment by Dikman** [comment]
> thanks for the report
> 
> are you using docker ? or native golang on your system ?
> which operating system are you using and in which shell do you start the server ?

I am using PS with Windows 10 and native GO. I do not use Docker.
```
D:\>go version
go version go1.15.7 windows/amd64
```

**Comment by mikeychowy** [comment]
> > thanks for the report
> > are you using docker ? or native golang on your system ?
> > which operating system are you using and in which shell do you start the server ?
> 
> I am using PS with Windows 10 and native GO. I do not use Docker.
> 
> ```
> D:\>go version
> go version go1.15.7 windows/amd64
> ```
> 
> IDE - VS Code 1.52.1.
> Shell - Bash 4.4.23.
> 
> ```
> $ bash --version
> GNU bash, version 4.4.23(1)-release (x86_64-pc-msys)
> Copyright (C) 2016 Free Software Foundation, Inc.
> License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
> 
> This is free software; you are free to change and redistribute 
> it.
> There is NO WARRANTY, to the extent permitted by law.
> ```

Windows have problems running prefork, i mostly use docker or wsl if i want to test out the prefork.

Better yet, maybe it's time to change os? Linux is immensely useful for development anyway.

**Comment by Dikman** [comment]
> Windows have problems running prefork, i mostly use docker or wsl if i want to test out the prefork.
> 
> Better yet, maybe it's time to change os? Linux is immensely useful for development anyway.

Maybe you are right, it needs to think about...
But v2.3.3 doesn't have this problem.

**Comment by kiyonlin** [comment]
Thanks for reporting, I will fix this asap.

**Comment by kiyonlin** [comment]
Released [v2.4.1](https://github.com/gofiber/fiber/releases/tag/v2.4.1)

**Comment by Dikman** [comment]
> Released [v2.4.1](https://github.com/gofiber/fiber/releases/tag/v2.4.1)

Thank you very much! This works :)
